components/inputs/input-styles.js

```diff
@@ -0,0 +1,72 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+
+export const inputStyles = css`
+	.d2l-input {
+		background-color: #ffffff;
+		border-color: var(--d2l-color-galena);
+		border-radius: 0.3rem;
+		border-style: solid;
+		border-width: 1px;
+		box-shadow: inset 0 2px 0 0 rgba(181, 189, 194, .2); /* corundum */
+		color: var(--d2l-color-ferrite);
+		box-sizing: border-box;
+		display: inline-block;
+		font-family: inherit;
```

Comment:
Since style mixins are gone, we can bring `font-family` back into the shared styles 🎉